### DISCUSSION TITLE
New Label: PRs Welcome

### DISCUSSION THREAD

**Discussion by Tyler-Keith-Thompson** [wwt/SwiftCurrent#45] [Ideas] [answered]
The team has a public board that's incredibly useful to see what you're working on and what is next up. That means as a consumer I've got regular access to the roadmap and know what I can expect. 

I suggest we add a "PRs Welcome" label that we can combine with others like "Good Issue" or potentially the proposed "Low Priority" to indicate to potential contributors that these items are things they can work on with no fear of interrupting work the team is currently doing. This means they don't have to go look at and understand the board, they know if they want to contribute here are the things that we'd welcome.

Of course we're willing to accept PRs for anything, this isn't meant to exclusively say "You can only work on these things" it's more of an invitation "We're working over here, wanna contribute? Take these issues and you probably won't conflict with our active work"

**Comment by Richard-Gist** [comment]
The way I'm thinking of this is:

We use "Help Wanted" when we have an issue that we want resolved, or someone is stuck on a PR and needs some assistance getting it through/developed.  For issues, this would align with something that is on our Roadmap, something we really want to complete, but we realize that we may not get to it soon and so help on the issue would help us achieve our goals.

We use "PRs Welcome" on issues that are probably low priority, or at least in areas that will not cause excessive conflicts for what we are currently doing.  They could be applied to anything that we would happily let the community contribute on.

+1 to the new label.

**Reply by Tyler-Keith-Thompson** [reply]
Yeah also "Help Wanted" can be for Non-PR things. For example I've seen repositories ask for maintainers with an issue and "Help Wanted" on it.

**Comment by Richard-Gist** [comment]
Possible meh description for the label:

```
Work here won't conflict with current dev team work
```

**Comment by morganzellers** [comment, marked as answer]
We decided to pass on adding this label for now. In the future if we feel the need for it after getting more community interest in contributing, we will revisit.
